src/lib/libcmd/cp.c

```diff
@@ -90,7 +90,7 @@ static const char usage_tail[] =
     "quit response (\bq\b or \bQ\b) exits immediately, and all other "
     "responses skip the file.]"
 "[r|R:recursive?Operate on the contents of directories recursively.]"
-"[s:symlink|symbolic-link?Make symbolic links to destination files.]"
+"[s:symlink|symbolic|symbolic-link?Make symbolic links to destination files.]"
```

Comment:
`--symbolic` already works because getopts completes abbreviates long options, so this addition is redundant.